app/views/pageflow/entries/show.html.erb

```diff
@@ -11,6 +11,7 @@
 
     <%= tag :link, :rel => 'icon', :href => image_path("pageflow/themes/#{@entry.theme.directory_name}/favicon.ico"), :type => 'image/ico' %>
     <%= meta_tags_for_entry(@entry) %>
+    <%= auto_discovery_link_tag(:atom, entries_url(format: 'atom')) if @theming %>
```

Comment:
This looks confusing to me. The context that `@theming` is only set if it could be determined based on the request is hard to get if you don't know the controller code. Maybe we can move this into a helper and make the `Theming.for_request` call there?

There would also be the alternative to use `Configuration#theming_url_options` the way [`pretty_entry_url`](https://github.com/codevise/pageflow/blob/6e500d5c821d06764ae6b4f4d52c6d717cf86465/app/helpers/pageflow/entries_helper.rb#L8) does. That way we could use the canonical entry url (including CNAME etc) even in cases like the preview where the request host name does not match the desired CNAME.